ocaml/xapi/xapi_vgpu.ml

```diff
@@ -66,12 +68,13 @@ let atomic_set_resident_on ~__context ~self ~value = assert false
 
 let copy ~__context ~vm vgpu =
   let all = Db.VGPU.get_record ~__context ~self:vgpu in
-  let vgpu = create ~__context
+  let vgpu = create' ~__context
       ~device:all.API.vGPU_device
       ~gPU_group:all.API.vGPU_GPU_group
       ~vM:vm
       ~other_config:all.API.vGPU_other_config
       ~_type:all.API.vGPU_type
+      ~expected_vm_state:(vm |> fun self -> Db.VM.get_power_state ~__context ~self)
```

Comment:
This is a little odd though: it essentially just bypasses the power-state check. How about having a parameter for just that instead? E.g. `~no_powerstate_check`.

Reply:
done: df49140bfdc1f12e8b344e8f1be3fa00c58f8e33